init.lua

```diff
@@ -117,6 +118,727 @@ food.module("lemon",function()
 	})
 end, true)
 
+-- Ice
+food.module("ice-bowl", function()
+	minetest.register_craftitem(":food:icebowl",{
+		description = S("Ice-Bowl"),
+		inventory_image = "food_sweet_ice_bowl.png",
+		groups = {food_bowl=1, food_ice=1}
```

Comment:
The new Bowl is an alternative to the existing Bowl. The Recipes for the Cups but should take any  of this Bowls ;-).